recipe/pre-7.3.6-0001.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/src/_c_internal_utils.c b/src/_c_internal_utils.c
```

Comment:
Do we want to apply this patch to upstream Matplotlib or is this really a transitory thing?

Reply:
Not sure, came in with #298. @mattip?

Reply:
This is needed until PyPy can make a new release with the required fix. That should happen in a month or two.

Reply:
It looks like it will do no harm in any case though?  What is the downside of pulling this upstream?

Reply:
I can see no harm in discarding the return value since it will always return `NULL` anyway. Perhaps some future version of Python will start returning a value at some point, but I doubt it since this is in error-handling code.